x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/flyout/index.tsx

```diff
@@ -210,7 +210,12 @@ export const EventFiltersFlyout: React.FC<EventFiltersFlyoutProps> = memo(
     );
 
     return (
-      <EuiFlyout size="l" onClose={handleOnCancel} data-test-subj="eventFiltersCreateEditFlyout">
+      <EuiFlyout
+        size="l"
+        onClose={handleOnCancel}
+        data-test-subj="eventFiltersCreateEditFlyout"
+        maskProps={{ style: 'z-index: 5000' }}
```

Comment:
I could not find this usage in the Eui docs (https://elastic.github.io/eui/#/utilities/overlay-mask), just want to make sure that the value to `style` here is a string rather than an object of type `React.CSSProperties` (an object with camelCased property names).

Also - can you confirm this component is displayed correctly on the "Endpoint > Event Filters" page (edit existing or create new should display it). Maybe include a screen capture in the PR's description.

Reply:
@paul-tavares sure, it appears to be 
![image](https://user-images.githubusercontent.com/56408403/156443316-7864be39-4efd-44dd-a80a-83bab67e4f9b.png)
will also add to description. Re: overlay mask, I will double check the docs/eui code to see, to be honest all I did was open the 2 components that were rendering above the timeline after looking at the one that didn't, and the reason was immediately obvious https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/common/components/exceptions/add_exception_flyout/index.tsx#L427 and I just copy and pasted it ha

Reply:
Thanks Kevin. I'm 👍 'it

Reply:
@paul-tavares actually i lied, the EuiSuperSelect options do not appear to be rendering correctly

Reply:
@kqualters-elastic so you are backing this change out?

Reply:
@paul-tavares no actually I fixed that by wrapping the event filter page in SecuritySolutionPageWrapper, which includes an important! css override that makes everything work https://github.com/elastic/kibana/pull/126195/commits/041e220a61cde925ca286512a5ad6cd56c1fc66d as long as y'all are ok with that change I'd like to still merge